### ISSUE TITLE
'swagger.json' file is not generated

### ISSUE THREAD

**Issue by azanorth** [Romakita/ts-express-decorators#323]
Type | Version
---|---
Question | 4.13.3

****
## Description
Hi @Romakita,
Could you tell me please why the 'swagger.json' file is not generated after the startup? It just does not appear in the directory. In previous versions it worked.

**Comment by Romakita** [comment]
@azanorth Yes I'd just identified the problem. I work on.

**Comment by azanorth** [comment]
Ok))

**Comment by Romakita** [comment]
Ok I remember why. This feature is bugged because it produce a lot of redundant info in the generated swagger.json.  
My proposal is to restore this feature with a new options in the swagger configuration.
```typescript
@ServerSettings({
   swagger: {
      specPath: './spec/swagger.base.json', // must be different to outFile.
      outFile: './spec/swagger.json
  }
})
```

Sorry for this error. I'll release the fix soon.
See you,
Romain

**Comment by azanorth** [comment]
Okay, thank you for your reply.

**Comment by Romakita** [comment]
fixed v4.14.4 :)

```typescript
@ServerSettings({
   swagger: {
      specPath: './spec/swagger.base.json', // must be different to outFile.
      outFile: './spec/swagger.json
  }
})
```

**Comment by azanorth** [comment]
This is great, everything works fine, thanks.
